src/Glpi/Form/QuestionType/QuestionTypesManager.php

```diff
@@ -159,4 +159,104 @@ protected function loadQuestionsTypes(): void
             fn(QuestionTypeInterface $a, QuestionTypeInterface $b) => $a->getWeight() <=> $b->getWeight()
         );
     }
+
+    public function getTemplateSelectionForCategories(): string
+    {
+        $icons = array_combine(
+            array_map(
+                fn(QuestionTypeCategory $type) => $type->value,
+                (array) $this->getCategories()
+            ),
+            array_map(
+                fn(QuestionTypeCategory $type) => $type->getIcon(),
+                (array) $this->getCategories()
+            )
+        );
+        $js_icons = json_encode($icons);
+
+        return <<<JS
+            function(item) {
+                const icons = {$js_icons};
+                return $('<span class="d-flex flex-row-reverse align-items-center gap-2">'
+                    + '<i class="' + icons[item.id] + '"></i>'
+                    + item.text
```

Comment:
I am not sure of this, but I guess that data is not yet sanitized at this point.
Please check with an item that have `<script>alert(1)</script>` in its name, or something like that.
```suggestion
                    + '<i class="' + _.escape(icons[item.id]) + '"></i>'
                    + _.escape(item.text)
```